plugins/ssh-keys/functions

```diff
@@ -12,10 +12,12 @@ list_ssh_keys() {
 verify_ssh_key_file() {
   declare desc="Test that public key is valid"
   [[ -s ${DOKKU_ROOT}/.ssh/authorized_keys ]] || dokku_log_fail "No public keys found."
-  local key line=0
+  local key line=0 tmp_key_file="$(mktemp)"
+  trap "rm \"$tmp_key_file\"" EXIT
```

Comment:
In many cases, yes. But a quick test shows that white-space in the `$tmp_key_file` will be interpreted incorrectly:

```
main() {
    t="/tmp/tmp file"
    touch "$t"
    trap "rm ""$t" EXIT
}
main
```
results in :
```
rm: cannot remove '/tmp/tmp': No such file or directory
rm: cannot remove 'file': No such file or directory
```

`"rm ""$t"` did not help and the lint fails with various single quotes as well, while `trap "rm "\""$t"\" EXIT` produces `SC2140: The double quotes around this do nothing. Remove or escape them.`

The only solution may be to define the trap clean as another function.

Thanks for your help.